### PR TITLE
User can retrieve post details of a specified Instagram user.

### PR DESCRIPTION
**Pull request by jyaaan**

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const store = require('./store');
 
 const UsernameInput = require('./input-username');
 const UserProfile = require('./user-profile');
+// const PostDetails = require('./post-details');
```
Comment (victormejia) [inline client/index.js]: don't need this?

File: server/ig-main.js [hunk 2]
```diff
@@ -17,17 +19,39 @@ IG.prototype.getAccountByName = function(username, session) {
...
+            cb();
+          }, GLOBAL_INTERVAL);
+        } else {
+          const endTime = new Date();
+          console.log('rate for this action (actions/minute):', counter/((endTime - startTime)/60000));
+          resolve(posts);
+        }
+      })
+    }
+    cb();
```
Comment (victormejia) [inline server/ig-main.js]: let's name this something other than `cb`, what is it actually doing?

File: server/index.js [hunk 3]
```diff
@@ -111,14 +105,14 @@ app.get('/rate-experiment', (req, res) => {
...
-          .then((detail) => {
-            console.log(detail._params);
-          })
-      })
+      // result.map((profile) => {
+      //   ig.getAccountById(profile.id, currentSession.session)
+      //     .then((detail) => {
+      //       console.log(detail._params);
+      //     })
+      // })
```
Comment (victormejia) [inline server/index.js]: delete unused code

File: server/scheduler.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+//
+// const MasterScheduler = () => {
+//
+// }
+//
+// const chainGetPosts = (userId, interval) => {
+//
+// }
+
+// In order to chain a function,
```
Comment (victormejia) [inline server/scheduler.js]: more uncommented code here :)

Comment (jyaaan) [inline server/scheduler.js]: Oop! I don't need scheduler.js anymore so I'll remove it.

**Pull request merged**